get.sh

```diff
@@ -75,7 +75,7 @@ function install_pvsadm() {
 
     if command -v "pvsadm" &> /dev/null; then
         echo "pvsadm is already installed!"
-        pvsadm version
+        pvsadm --version
```

Comment:
this will be problem because it tries with the existing command which doesn't this functionality, lets use the `pvsadm version` itself for timebeing, will remove later..

Reply:
Agree. I've added a TODO to revisit this section later in time. 

Thanks! 😃